docs/Components.md

```diff
@@ -955,50 +955,50 @@ Read more about [peripherals](Supported-Peripherals.md).
+| 7750 | Switch_d7      | Switch with internal pull-down resistor (Active High)                                                   |
+| 7751 | Switch_d8      | Switch with internal pull-down resistor (Active High)                                                   |
+| 7752 | Switch_d9      | Switch with internal pull-down resistor (Active High)                                                   |
+| 7753 | Switch_d10     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7754 | Switch_d11     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7755 | Switch_d12     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7756 | Switch_d13     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7757 | Switch_d14     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7758 | Switch_d15     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7759 | Switch_d16     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7760 | Switch_d17     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7761 | Switch_d18     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7762 | Switch_d19     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7763 | Switch_d20     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7764 | Switch_d21     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7765 | Switch_d22     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7766 | Switch_d23     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7767 | Switch_d24     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7768 | Switch_d25     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7769 | Switch_d26     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7770 | Switch_d27     | Switch with internal pull-down resistor (Active High)                                                   |
+| 7771 | Switch_d28     | Switch with internal pull-down resistor (Active High)                                                   |
```

Comment:
I would not give any information about active high vs low here because it all depends on SwitchMode

Reply:
ok swapped accordingly